kalite/distributed/views.py

```diff
@@ -212,6 +213,21 @@ def search(request):
         'category': category,
     }
 
+def create_superuser(request):
```

Comment:
I don't know how we feel about TastyPie these days (@rtibbles, @jamalex), but given that `User` is a Model it should be really easy to create a `UserResource` that handles this kind of thing automatically.